onnxruntime/core/graph/graph_utils.cc

```diff
@@ -399,9 +399,24 @@ bool IsGraphInput(const Graph& graph, const NodeArg* input) {
   return std::find(graph_inputs.begin(), graph_inputs.end(), input) != graph_inputs.end();
 }
 
+bool IsConstantInitializer(const Graph& graph, const std::string& initializer_name) {
```

Comment:
I think we can/should merge IsConstantInitializer and NodeArgIsConstant, i.e., make the one call the other, since they are doing a very similar thing.

Reply:
One is comparing pointers, one is comparing names. I left both as the pointer comparison should be faster, but maybe we don't care about such a performance difference at this point. 

---
In reply to: [299653968](https://github.com/microsoft/onnxruntime/pull/1324#discussion_r299653968) [](ancestors = 299653968)